errata/models.py

```diff
@@ -28,14 +28,16 @@
 
 NEW = 'New'
 EDITORIAL_REVIEW = 'Editorial Review'
-ANDREW_EDITORIAL_REVIEW = 'Andrew Editorial Review'
+K12_EDITORIAL_REVIEW = 'K-12 Editorial Review'
+KELSEY_EDITORIAL_REVIEW = 'Kelsey Editorial Review'
 ANTHONY_EDITORIAL_REVIEW = 'Anthony Editorial Review'
 REVIEWED = 'Reviewed'
 COMPLETED = 'Completed'
 ERRATA_STATUS = (
     (NEW, 'New'),
     (EDITORIAL_REVIEW, 'Editorial Review'),
-    (ANDREW_EDITORIAL_REVIEW, 'Andrew Editorial Review'),
+    (K12_EDITORIAL_REVIEW, 'K-12 Editorial Review'),
```

Comment:
changing the value of the database stored value can cause data problems - I'd recommend against this.

Also, related, perhaps don't put peoples names in the database `KELSEY_EDITORIAL_REVIEW = 'Kelsey Editorial Review'` to `CUSTOM_EDITORIAL_REVIEW = 'Custom Editorial Review'` or something like that